mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### Breaking changes
+
+* Disallow the usage of `--incremental-upload` command-line argument on non S3 Express buckets ([#1297](https://github.com/awslabs/mountpoint-s3/pull/1297)).
```

Comment:
very nitpick: the period should be before the PR link to be consistent with prev changelog.

(only putting this since we need to make the other change)